ci/build.sh

```diff
@@ -1,5 +1,36 @@
 #!/bin/bash
 
+while getopts ":ahp:" opt; do
+  case $opt in
+    a)
+      echo "Build firmware images for all platforms"
+      TEMP=""
```

Comment:
Instead of TEMP, it'd be better to use PLATFORMS or PLATFORM, as TEMP could be confused with a temporary directory